grammar.js

```diff
@@ -100,16 +100,19 @@ module.exports = grammar({
 
     expression: $ => seq(
       '{',
-      $.expression_value,
-      '}'
+      prec.left(
+        seq(
+          alias(repeat($._expression_value), $.expression_value),
+          '}'
+        )
+      )
```

Comment:
I like this approach a lot! I'll review the Surface grammar and see if it has the same bug or not. If so I'll port this fix over.